tests/unit/lms/views/index_test.py

```diff
@@ -0,0 +1,8 @@
+from lms.views.index import index
+
+
+class TestIndexView:
+    def test_it_does_nowt(self, pyramid_request):
+        template_params = index(pyramid_request)
+
+        assert template_params == {}
```

Comment:
If you wanted, this could just be:

```python
    def test_it_does_nowt(self, pyramid_request):
        assert index(pyramid_request) == {}
```

Reply:
Yep, but in general in all of the other tests we tend to assign to a variable and then test that variable. I assumed this was for debugging purposes.